keys_test.go

```diff
@@ -36,22 +36,51 @@ func TestClient_CreateKey(t *testing.T) {
 
 	server.ResponseBody = expected
 
-	actual, err := client.Keys().Create(context.Background(), CreateKeyRequest{
+	actual, err := client.Keys().CreateAuthKey(context.Background(), CreateAuthKeyRequest{
 		Capabilities: capabilities,
 	})
 	assert.NoError(t, err)
 	assert.EqualValues(t, expected, actual)
 	assert.Equal(t, http.MethodPost, server.Method)
 	assert.Equal(t, "/api/v2/tailnet/example.com/keys", server.Path)
 
-	var actualReq CreateKeyRequest
+	var actualReq CreateAuthKeyRequest
 	assert.NoError(t, json.Unmarshal(server.Body.Bytes(), &actualReq))
 	assert.EqualValues(t, capabilities, actualReq.Capabilities)
 	assert.EqualValues(t, 0, actualReq.ExpirySeconds)
 	assert.EqualValues(t, "", actualReq.Description)
 }
 
-func TestClient_CreateKeyWithExpirySeconds(t *testing.T) {
+func TestClient_CreateOAuthClient(t *testing.T) {
```

Comment:
Nit - I would add this new test after the existing `CreateKey...` tests just to keep them grouped.